packages/react-conformance/src/defaultErrorMessages.tsx

```diff
@@ -658,4 +661,21 @@ export const defaultErrorMessages = {
       error,
     });
   },
+
+  'primary-slot-gets-native-props': (testInfo: IsConformantOptions, error: Error) => {
+    const { displayName } = testInfo;
+    const { resolveInfo } = errorMessageColors;
+
+    return getErrorMessage({
+      displayName,
+      overview: `doesn't forward native props (except className and style) to the primary slot.`,
```

Comment:
This is a little confusing with double negatives. Maybe something like this?
```suggestion
      overview: `doesn't properly pass native props to the primary and root slots.`,
```